metadata-io/src/main/java/com/linkedin/metadata/search/utils/SearchUtils.java

```diff
@@ -136,10 +137,16 @@ public static AggregationMetadata merge(AggregationMetadata one, AggregationMeta
     Map<String, Long> mergedMap =
         Stream.concat(one.getAggregations().entrySet().stream(), two.getAggregations().entrySet().stream())
             .collect(Collectors.toMap(Map.Entry::getKey, Map.Entry::getValue, Long::sum));
+
```

Comment:
Are there any unit tests for this static class? If not we should at least log some backlog ticket about the coverage here.